### PR TITLE
[Model Update] Digital product passport

### LINKED ISSUE #529: [Model Update]: Digital Product Passport
## Update Reason

- Conformity with industry core
- delete either-or
- minor update on description of the aspect itself

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists
- [x] The potential updates are discussed with all stakeholders
- [x] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [x] The potential update will be either 
  - [x] backward compatible or 
  - [ ] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by catroest**
## Description
Updates sonform with the latest provisonal agreement or the DIgital Product Passport Regulation:
https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CONSIL:ST_5147_2024_INIT

Overview (internal): https://confluence.catena-x.net/display/CDPP/Model%3A+Digital+Product+Pass

 -->

Closes #529 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.5.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

**Comment by catroest** [comment]
@agg3fe please check out for MS2 :)

**Comment by catroest** [comment]
validation fails, as #603 should be merged first

**Comment by agg3fe** [comment]
@catroest could you please also update your branch with main branch.

File: io.catenax.generic.digital_product_passport/4.0.0/DigitalProductPassport.ttl [hunk 1]
```diff
@@ -0,0 +1,1167 @@
...
+   samm:dataType :ProductFootprintEntity .
+
+:SourceCategoryEnumeration a samm-c:Enumeration ;
+   samm:preferredName "Source Category Enumeration"@en ;
+   samm:description "The types of sources that can be linked in a digital product passport can vary depending on the nature of the product and the information that needs to be included. This is an enumeration of possible sources."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Product Specifications" "Manufacturer Information" "User Manuals and Guides" "Certifications and Compliance" "Product Images and Videos" "Warranty Information" "Reviews and Ratings" "Product Variations" "Supply Chain Information" "Environmental Impact" "Compatibility and Accessories" "FAQs and Support" "Purchase and Retail Information" "Privacy and Data Handling" "Third-Party Integrations" "Legal Information" "Safety Information" "Repair and Installation" "Waste Generation and Prevention" "Specific Voluntary Labels" "Product Packaging" "Return and Disposal" "End of Life" "Material and Substance Information" "Technical Documentation" "Treatment facilities" "Other" ) .
+
+:HeaderTrait a samm-c:Trait ;
```
Comment (agg3fe) [inline io.catenax.generic.digital_product_passport/4.0.0/DigitalProductPassport.ttl:501]: missing `preferredName` and `description`

File: io.catenax.generic.digital_product_passport/4.0.0/DigitalProductPassport.ttl [hunk 2]
```diff
@@ -0,0 +1,1167 @@
...
+   samm:characteristic :FootprintList .
+
+:materialFootprint a samm:Property ;
+   samm:preferredName "Material Footprint"@en ;
+   samm:description "The matrial footprint of the product. This is mentioned in the ESPR provisional agreement from January 9th, 2024 Annex I:\n(ma) the material footprint of the product;\nand defined by Article 2:\n(25a) 'material footprint' refers to the total amount of raw materials extracted to meet final consumption demands."@en ;
```
Comment (agg3fe) [inline io.catenax.generic.digital_product_passport/4.0.0/DigitalProductPassport.ttl:722]: typo `matrial`

File: io.catenax.generic.digital_product_passport/4.0.0/DigitalProductPassport.ttl [hunk 3]
```diff
@@ -0,0 +1,1167 @@
...
+   samm:preferredName "Operation"@en ;
+   samm:description "Operational information of the product."@en ;
+   samm:characteristic :OperationCharacteristic .
+
+:handling a samm:Property ;
+   samm:preferredName "handling"@en ;
+   samm:description "Properties connected with the handling of the product."@en ;
+   samm:characteristic :HandlingCharacteristic .
+
+:characteristics a samm:Property ;
```
Comment (agg3fe) [inline io.catenax.generic.digital_product_passport/4.0.0/DigitalProductPassport.ttl:69]: Property and the referenced Characteristic should not have the same name

**Review by agg3fe — CHANGES_REQUESTED** [review]
please check my comments

**Review by agg3fe — APPROVED** [review]
LGTM

**Comment by catroest** [comment]
@agg3fe the two commits are not MS2 relevant:
- the first one adds 3 missing example values
- the second one changes mistakes in the description where ' was translated wrongly to ?

**Pull request merged**